lib/file_transfer_agent/gcs_util.js

```diff
@@ -37,15 +41,24 @@ function GCSLocation(bucketName, path) {
 /**
  * Creates an GCS utility object.
  *
- * @param {module} httpclient
- * @param {module} filestream
+ * @param {module} httpClient
+ * @param {module} fileStream
```

Comment:
the first parameter is missing in documentation

Reply:
Fixed.